README.md

```diff
@@ -7,8 +7,8 @@
 
 <div class="flex-container" align="center">
     <div class="flex-container" align="center">
-        <a href="https://img.shields.io/badge/Python-3.7%7C3.8%7C3.9%7C3.10-blue">
-        <img src="https://img.shields.io/badge/Python-3.7%7C3.8%7C3.9%7C3.10-blue"
+        <a href="https://img.shields.io/badge/Python-3.9%7C3.10%7C3.11%7C3.12%7C3.13-blue">
```

Comment:
@kipkurui removed 3.7 (it was awful anyway) and removed 3.8. Technically 3.8 should work and its the first real arm64 support, but there would be too many edge-cases to potentially support